app/lib/actions/katello/applicability/hosts/bulk_generate.rb

```diff
@@ -32,8 +32,8 @@ def hostname(host_id)
           end
 
           def humanized_name
-            if input[:host_ids]&.length == 1
-              _("Bulk generate applicability for host %s" % hostname(input[:host_ids].first))
+            if input && input[:host_ids]&.length == 1
```

Comment:
Was getting a null error during translation. I guess the translations for action humanized name runs with null input causing the issue. It's a harmless check so committed this.

Reply:
This was the error:

```
NoMethodError: undefined method `[]' for nil:NilClass
/home/vagrant/katello/app/lib/actions/katello/applicability/hosts/bulk_generate.rb:35:in `humanized_name'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/foreman-tasks-7.0.0/app/lib/actions/entry_action.rb:56:in `block in all_action_names'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/foreman-tasks-7.0.0/app/lib/actions/entry_action.rb:56:in `map'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/foreman-tasks-7.0.0/app/lib/actions/entry_action.rb:56:in `all_action_names'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/foreman-tasks-7.0.0/lib/tasks/gettext.rake:16:in `block (2 levels) in <main>'
/home/vagrant/foreman/lib/tasks/gettext.rake:45:in `block in <main>'
```